### PR TITLE
Configurable rocksdb cache size

### PR DESCRIPTION
**Pull request by NikVolf**

### CODE REVIEW

File: parity/cli.rs [hunk 1]
```diff
@@ -155,6 +155,7 @@ Footprint Options:
   --cache MEGABYTES        Set total amount of discretionary memory to use for
                            the entire system, overrides other cache and queue
                            options.
+  --db-cache-size MEGABYTE Database cache size.
```
Comment (arkpar) [inline parity/cli.rs]: Perhaps this should be a single  --cache options with 20% going to blockchain cache and 80% to db cache, actual ratio needs research though

Comment (rphmeier) [inline parity/cli.rs]: The ratio could also be configurable in that case.

Comment (NikVolf) [inline parity/cli.rs]: probably later if ratio is not known

**Comment by NikVolf** [comment]
for some reason travis does not run

**Comment by rphmeier** [comment]
Seems to be in the queue: https://travis-ci.org/ethcore/parity/builds/138874746

**Pull request merged**